Dockerfile

```diff
@@ -85,8 +85,7 @@ RUN poetry install --no-dev -vvv && \
 ENV NODE_ENV=production
 RUN npm run build && \
     npm run build-scss && \
-    rm -rf node_modules/ && \
-    apt-get remove --purge --auto-remove -y ca-certificates
+    rm -rf node_modules/
```

Comment:
Maybe also add a comment saying that you used to remove ca-certificates but no longer do and explain why so no one tries the experiment again?